pages/cc_copy_page.php

```diff
@@ -42,7 +42,11 @@
         <option value="" disabled selected hidden>Please select an external module...</option>
     <?php
     foreach($module_mapping as $prefix => $key) {
-        echo "<option value='$key' prefix='$prefix'>" . \ExternalModules\ExternalModules::getConfig($prefix)['name'] . "</option>";
+        // Check if a configuration exists because the query results can contain disabled EM's. See comment in ExternalModule->collectModules for clarification.
+        $config = \ExternalModules\ExternalModules::getConfig($prefix)['name'];
+        if(isset($config)){
+            echo "<option value='$key' prefix='$prefix'>" . $config . "</option>";
+        }   
```

Comment:
This doesn't appear to actually change anything thanks to your alteration to `collectModules`, which makes me wonder if I'm missing some cases in my local setup.

Either way I don't see any harm in keeping it, it certainly works when I rollback the correction to `collectModules`.